dispatcher/filtering.py

```
@@ -0,0 +1,81 @@
+import requests
+
+
+def process_selector(
+    selector: dict[str, str],
+    endpoint: str,
+    token: str,
+    selector_mapping: dict[str, str] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)